### ISSUE TITLE
toolbelt bug

### ISSUE THREAD

**Issue by Timonkeyn** [tgstation/tgstation#44762]
Issue reported from  Round ID: 111332 (/tg/Station Bagil [ENGLISH] [US-WEST] [100% FREE LAG])

Reporting client version: 512.1471

**Comment by HomocidlePancakes** [comment]
Ugh... Where is the bug?

**Comment by Timonkeyn** [comment]
well i pressed enter too fast, so here is the Story:

i was looting tool storage and took a belt at the same time as another Person, Terry Robinson. i quickly put it on with E and wondered why it filled up, then it teleported to my Hand and back to my waist. then i realised i couldnt interact with it and that it was now stuck to my waist. i examined myself, and it was in fact, there. i tried to put another belt on and it didnt work. i realised it was someone elses belt when i saw the items moving. i messaged Terry and tried to convince him that i wasnt lying. he went into space and i ask Tlal (Brick Carr) to take the toolbelt off. it then disappeared from the other persons waist as it was now mine. later he asked me to put the toolbelt back on and now it reappeared for that other Person.

**Comment by Timonkeyn** [comment]
My Name was Kobe Catster

**Comment by HomocidlePancakes** [comment]
Oh, the lag dupe. It works on all the items, but good luck pulling it off intentionally.

**Comment by Timonkeyn** [comment]
Round ID: 111334

okay it just fucking happened to me again but with a Flashlight

it went from my Right pocket onto my belt Slot, had to ask Brick again to take it off, then it disappeared from my Right pocket AGAIN and brick had it, but there already was something in my Right pocket. i asked Pepper Oni to take it out and IT WAS A FUCKING FLASHLIGHT. IT WAS DUPLICATED. (i couldnt put anything into my Right pocket and there wasnt anything inside)

(i wrote this while 111334 was still going and i probably posted this after the round has finished)
TL;DR Extradimensional items and item duplicating

**Comment by GravitronBlue** [comment]
Sounds similar to #41363 given the nature of item interaction, specifically in this case where an item being carried by another player can still be interacted with (picked up) by others.

Sounds like there's also some odd inventory HUD crossover code stuff that can still occur, should that happen try clicking on the missing/ghost item on your HUD while near the person who has it to steal it back.
![Thosearemyinsuls](https://user-images.githubusercontent.com/17460297/59993052-91ff3280-9603-11e9-8529-85520702cf30.PNG)

**Comment by nfreader** [comment]
As per oranges, this is a common race condition bug in inventory that hasn't been fixed yet.

**Comment by timothymtorres** [comment]
This was likely fixed with #52115
